packages/components/src/Card/Card.tsx

```diff
@@ -1,26 +1,57 @@
-import React, { ReactNode } from "react";
+import React, { ReactElement, ReactNode } from "react";
 import classnames from "classnames";
 import { XOR } from "ts-xor";
 import styles from "./Card.css";
 import colors from "./colors.css";
 import { CardClickable } from "./CardClickable";
+import { Button, ButtonProps } from "../Button";
 import { Heading } from "../Heading";
 
-interface CardProps {
+interface CardCommonProps {
   /**
    * The `accent`, if provided, will effect the color accent at the top of
    * the card.
    */
   readonly accent?: keyof typeof colors;
   readonly children: ReactNode | ReactNode[];
+}
 
+interface HeaderActionProps {
   /**
    * The title of the card.
    */
   readonly title?: string;
+  /**
+   * The action props that renders into a button on the card header.
+   */
+  readonly action?: Omit<ButtonProps, "size" | "fullWidth">;
+}
+
+type HeaderProps = string | HeaderActionProps | ReactElement;
+
+interface CardProps {
+  /**
+   * The `accent`, if provided, will effect the color accent at the top of
+   * the card.
+   */
+  readonly accent?: keyof typeof colors;
+  readonly children: ReactNode | ReactNode[];
+  /**
+   * @deprecated
+   * The title of the card. Use header instead.
+   *
```

Comment:
get it straight to the point

```suggestion
   * @deprecated - Use header instead.
```

Reply:
straigtened